docs/command/atlas-kubernetes-config-generate.txt

```diff
@@ -56,7 +56,7 @@ Options
    * - --operatorVersion
      - string
      - false
-     - Version of Atlas Kubernetes Operator to generate resources for. This value defaults to "2.3.0".
+     - Version of Atlas Kubernetes Operator to generate resources for. This value defaults to "2.2.0".
```

Comment:
AKO 2.3 has not been (fully) released yet, initially, I planned to have hidden support for streams but it is overly complicated to keep this feature matrix without breaking things. So I rolled back the "latest supported version" and kept the code which won't run until we finish the release and have a green light to support 2.3 stuff on CLI.

I added a compatibility check https://github.com/mongodb/mongodb-atlas-cli/pull/2967/files#diff-f4548e3bffdbb9c6ad62fdc079f212e3a6cf849c64b43cedd44ece44790a35d6R354-R358 to avoid execute 2.3 feature to be executed. Once we bump this again https://github.com/mongodb/mongodb-atlas-cli/pull/2967/files#diff-2da3e905a9e0b334a8a50b432a3dce0a702e75bd37c3df058c3589fd31c7a964R30 the check will pass and 2.3 feature will be available.

As I already removed hard dependency on the required resources to execute k8s tests, we won't break on future releases and can new features without breaking changes (if the feature flag is used properly)